stylesheets/style-about.css

```
@@ -0,0 +1,175 @@
+/* 242F9B  646FD4  9BA3EB   DBDFFD */
+
+img {
+    height: 40vh;
+    object-fit: cover;
+    width: 40vw;
+} 
+
+h1 {
+    padding-bottom: 10px;
+}
+
+h3 {
+    font-size: 1.5em;
+    padding-left: 20px;
+    line-height:0%;
+}
+
+#container-about {
+    display: grid;
+    grid-template-rows: .2fr .75fr .15fr .3fr .15fr .75fr .1fr;
```

Oooh, very fancy.  :)

I'd be inclined to use integers here instead of fractions though:

```suggestion
    grid-template-rows: 20fr 75fr 15fr 3fr 15fr 75fr 10fr;
```

It has the same effect but is a little shorter and I think is a little easier to read.